timon/plugins/http_server/views.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+# #############################################################################
+# Copyright : (C) 2023 by MHComm. All rights reserved
+#
+# __author__ = "Quentin Laymajoux"
+# __email__ = "info@mhcomm.fr"
+#
+# Name       : timon.plugins.http_server.views
+"""
+Summary: The http server plugin views and routes
```

Comment:
At first I did not understand what you mean, but I guess you mean.


"The http server plugin's views and routes"

Reply:
done